packages/core/src/components/non-ideal-state/nonIdealState.tsx

```diff
@@ -14,22 +14,22 @@ import { IProps } from "../../common/props";
 
 export interface INonIdealStateProps extends IProps {
     /*
-     * An action the user can take to correct the non-ideal state.
+     * An action that's attached to the non-ideal state.
```

Comment:
What does correcting a non-ideal state mean? It may or may not need correction, I'd rather not assume a particular usage here